### PR TITLE
Adopt sbt-clue

### PR DESCRIPTION
**Pull request by armanbilge**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -55,47 +53,23 @@ val modelTests =
...
-        Def.task {
-          (templates / Compile / scalafix)
-            .toTask(s" GraphQLGen --out-from=$outFrom --out-to=$outTo")
-            .value
-          (to ** "*.scala").get
-        }
-      }.taskValue,
-      // Include schema files from templates in jar.
-      Compile / unmanagedResourceDirectories += (templates / Compile / resourceDirectory).value
+      Compile / clueSourceDirectory := (ThisBuild / baseDirectory).value / "lucuma-schemas" / "src" / "clue",
```
Comment (rpiaggio) [inline build.sbt:68]: Why is this necessary?

File: build.sbt [hunk 2]
```diff
@@ -55,47 +53,23 @@ val modelTests =
...
+      // Include schema files in jar.
+      Compile / unmanagedResourceDirectories += (Compile / clueSourceDirectory).value / "resources"
```
Comment (rpiaggio) [inline build.sbt:70]: Ah, I guess it's just to reference it here. Can we just put the schema file somewhere else?

**Review by rpiaggio — APPROVED** [review]
LGTM

File: build.sbt (hunk 2 above)
Comment (rpiaggio) [reply build.sbt:70]: Not sure when this is used. The schema files are read by the generator from the directories specified in `scalafix.conf` in the property `GraphQLGen.schemaDirs`.

Comment (armanbilge) [reply build.sbt:70]: Oh, I thought that downstreams needed this, so they can read it out of the resources in the Jar. is that not the case?

Comment (rpiaggio) [reply build.sbt:70]: Ah, yes, sorry. It has to be included in the Jar indeed. And yes, it makes sense to automatically include it. Do you think it's possible to automatically add it when invoking `scalafix GraphQLGen` so that users don't have to manually add it to `scalafix.conf`?

Comment (armanbilge) [reply build.sbt:70]: > Do you think it's possible to automatically add it when invoking

Oh yeah, that would be nice as well. Not sure if we can pass arbitrary config when invoking scalafix but maybe we can backchannel it somehow 🤔
